docs/content/parser/RunningSimulations.md

```diff
@@ -53,6 +32,29 @@ particles:
 
 ```
 
+## Running ABLATE from the Command Line
+If you built ABLATE using the command line you can run ABLATE using either the debug or release builds.
+    
+    ```bash
+    # navigate to either the debug or release directory of ABLATE
+    ./ablate --input /path/to/the/inputfile.yaml
+
+    # similarly, ABLATE can be run using mpi
+    mpirun -n 3 ./abalte --input /path/to/the/inputfile.yaml
```

Comment:
ablate is misspelled

Reply:
Thanks!  That has been in there for a while